features/admin.impersonation.v1/pages/impersonation-configuration.tsx

```diff
@@ -0,0 +1,277 @@
+                text: t("governanceConnectors:goBackLoginAndRegistration")
+            } }
+            bottomMargin={ false }
+            contentTopMargin={ false }
+            pageHeaderMaxWidth={ true }
+            data-componentid={ `${ componentId }-form-layout` }
+        >
+            <Ref innerRef={ pageContextRef }>
+                <Grid className={ "mt-2" } >
+                    <Grid.Row columns={ 1 }>
+                        <Grid.Column width={ 16 }>
+                            <EmphasizedSegment className="form-wrapper" padded={ "very" }>
+                                { isImpersonationFetchRequestLoading
+                                    ? renderLoadingPlaceholder()
+                                    : (
+                                        <>
+                                            <Form
+                                                id={ FORM_ID }
+                                                uncontrolledForm={ true }
+                                                onSubmit={ null }
+                                                initialValues={ impersonationConfig }
+                                                enableReinitialize={ true }
+                                                ref={ formRef }
+                                                noValidate={ true }
+                                                autoComplete="new-password"
+                                            >
+                                                <Grid>
+                                                    <Grid.Row columns={ 1 } key={ 1 }>
+                                                        <Grid.Column key="enableEmailNotification">
+                                                            <Field.Checkbox
+                                                                ariaLabel="Enable Email Notification"
+                                                                name="enableEmailNotification"
+                                                                label={ t("impersonation:form." +
+                                                                    "enableEmailNotification.label") }
+                                                                readOnly={ isReadOnly }
+                                                                width={ 16 }
+                                                                data-componentid={
+                                                                    `${componentId}-enable-email-notification` }
+                                                                toggle
+                                                                listen={ (value: boolean) => {
+                                                                    handleSubmit(value);
+                                                                } }
+                                                            />
```

Comment:
Let's also add a hint below this toggle, explaining the purpose and effects this config would cause